alwayson/controllers/injectors.py

```diff
@@ -0,0 +1,9 @@
+from alwayson.config import Config
+from alwayson.db.manifest_repo import ManifestRepo
+
+def inject_manifest_repo():
+  cfg = Config.create()
+  # TODO In middleware we would need to figure out how to get the
```

Comment:
My jwt experience is rusty - is that something that a nefarious user could modify and start hitting another customer's db?

Reply:
Everything that would be stored in the JWT would be test data, so I'm terribly worried but it would be good to ensure auth headers and tokens never get included in these JWTs